python/python/Ice/Future.py

```diff
@@ -183,3 +183,5 @@ def _warn(self, msg):
     StateRunning = "running"
     StateCancelled = "cancelled"
     StateDone = "done"
+
+__all__ = ["Future", "wrap_future", "FutureBase"]
```

Comment:
The __all__ variable specifies the public API and controls what is exposed when using the import * syntax. This prevents duplication warnings during documentation generation by avoiding the re-export of imported items, which would otherwise result in warnings indicating they are defined in multiple places.